vizro-core/docs/pages/user-guides/layouts.md

```diff
@@ -403,10 +413,329 @@ By default, the grid fits all charts/components on the screen. This can lead to
+
+!!! example "Flex - advanced example"
+    === "app.py"
+        ```{.python pycafe-link}
+        import vizro.models as vm
+        from vizro import Vizro
+        import vizro.plotly.express as px
+
+        tips = px.data.tips()
+
+        page = vm.Page(
+            title="Flex - advanced example",
+            layout=vm.Flex(direction="row", gap="40px", wrap=True),
+            components=[vm.Graph(figure=px.violin(tips, y="tip", x="day", color="day")) for i in range(3)],
+        )
+
+        dashboard = vm.Dashboard(pages=[page])
+        Vizro().build(dashboard).run()
+        ```
+
+    === "app.yaml"
+        ```yaml
+        # Still requires a .py to add data to the data manager and parse YAML configuration
+        # See yaml_version example
+        pages:
+          - components:
+              - figure:
+                  _target_: violin
+                  x: day
+                  y: tip
+                  color: day
+                  data_frame: tips
+                type: graph
+              - figure:
+                  _target_: violin
+                  x: day
+                  y: tip
+                  color: day
+                  data_frame: tips
+                type: graph
+              - figure:
+                  _target_: violin
+                  x: day
+                  y: tip
+                  color: day
+                  data_frame: tips
+                type: graph
+            layout:
+              type: flex
+              direction: row
+              gap: 40px
+              wrap: true
+            title: Flex - advanced example
+        ```
+
+    === "Result"
+        [![FlexAdvanced]][flexadvanced]
+
+### Change the size of flex items
+
+There may be times when resizing your flex items is necessary, such as when designing for different screen sizes or combining elements with diverse content types (e.g. charts, text, or images).
+
+You can achieve this by:
+
+- [either specifying the `width` and `height` directly on the components](#change-size-via-components-arguments)
+- [or by applying custom CSS](#change-size-via-custom-css)
+
+#### Change size via component's arguments
+
+The syntax for setting `width` and `height` varies between components. Refer to the component's documentation for the correct syntax and usage, such as:
+
+- `Graph`: See the documentation on [Plotly - Adjust Graph Size in Python](https://plotly.com/python/setting-graph-size/).
+- `AgGrid`: See the documentation on [Dash - Change Grid Size](https://dash.plotly.com/dash-ag-grid/grid-size)
+- `DataTable`: See the documentation on [Dash - Setting Table Height](https://dash.plotly.com/datatable/height#setting-table-height-with-vertical-scroll)
+- `Card`: See our documentation on [Card - The extra argument](https://vizro.readthedocs.io/en/stable/pages/user-guides/card/#the-extra-argument)
+
+!!! example "Change the width for Graph"
```

Comment:
Also this example. Maybe highlight the line that's different and emphasise you've set `width=400` and explain the difference that's made. Otherwise it's not really clear how it's different form the above or what it's demonstrating.